mdpow/templates/bar_opls.mdp

```diff
@@ -110,6 +110,7 @@ lincs-iter               = 1
 
 ; Free energy control stuff
 free_energy              = yes
+separate-dhdl-file       = no
```

Comment:
Added a test that checks that the edr is produced. I check the mdps produced and the contents of the edr manually with `gmx dump -e md.edr`; it all seems correct to me